### PR TITLE
Introduce timestamp transformers

### LINKED ISSUE #122: Timestamps feature + tsconfig question
Hi!

At first a big thank you for your work on this package, we are using it since a few month at work, and we love the way we can use typescript and not only json to seed our databases.

It could be great to have these features :
* add a transformer to set timestamps (createdAt, updatedAt)
* add CLI option to set the timestamps (based on the previous transformer)

I can work on this PR if you are OK with that ?

Also, maybe I missed something, but I would like to use my typescripts models inside data typescript files.

I did it and it's working well except when some models are using imports with paths, is there a way to specificy which tsconfig file to use when use CLI ?
Thanks

### PR DESCRIPTION
**Pull request by benjaminb-**
Hey @pkosiec, totally forgot to finalize the PR for the timestamp feature we discussed in May : https://github.com/pkosiec/mongo-seeding/issues/122

Can you review please, and tell me if you need me to add something else ?

Thanks :+1: 

Resolves #122

### CODE REVIEW

File: core/test/unit/transformer.ts [hunk 1]
```diff
@@ -57,4 +57,45 @@ describe('CollectionTransformer', () => {
...
+            name: 'Test 2',
+            value: 2,
+            createdAt: now,
+            updatedAt: now,
+          },
+        ],
+      },
+    ];
+
+    const actualData = collectionTransformer.transform(collections, [
```
Comment (pkosiec) [inline core/test/unit/transformer.ts:95]: To make sure the test is predictable even on slower machines, please mock the Date constructor using `jest.spyOn`, like this: https://stackoverflow.com/a/57599680

File: docker-image/.ci/script.sh [hunk 2]
```diff
@@ -18,7 +18,7 @@ TESTER_IMAGE_NAME="tester:latest"
 DB_NAME=dockertestdb
 
 # Import data
-docker run --rm -it --network="host" -e DB_NAME=${DB_NAME} -e REPLACE_ID=true -v ${DATA_IMPORT_PATH}:${DATA_IMPORT_PATH} -w ${DATA_IMPORT_PATH} $DOCKER_IMAGE_REPOSITORY:$CI_BUILD_NUMBER
+docker run --rm -it --network="host" -e DB_NAME=${DB_NAME} -e REPLACE_ID=true SET_TIMESTAMPS=true -v ${DATA_IMPORT_PATH}:${DATA_IMPORT_PATH} -w ${DATA_IMPORT_PATH} $DOCKER_IMAGE_REPOSITORY:$CI_BUILD_NUMBER
```
Comment (pkosiec) [inline docker-image/.ci/script.sh:21]: Probably that's why the Docker image tests failed on CI 🙂 
```suggestion
docker run --rm -it --network="host" -e DB_NAME=${DB_NAME} -e REPLACE_ID=true -e SET_TIMESTAMPS=true -v ${DATA_IMPORT_PATH}:${DATA_IMPORT_PATH} -w ${DATA_IMPORT_PATH} $DOCKER_IMAGE_REPOSITORY:$CI_BUILD_NUMBER
```

**Review by pkosiec — COMMENTED** [review]
Hi @benjaminb-,
Thanks a lot for your contribution! I put a few comments regarding your PR.

**Comment by benjaminb-** [comment]
Hi! Thanks for the review! I almost fix everything, still have to work a little bit on the integration testing.

File: cli/README.md [hunk 3]
```diff
@@ -84,6 +85,7 @@ You can use the following environmental variables while using `seed` tool:
 | DROP_DATABASE | `false` | Drops entire database before data import |
 | DROP_COLLECTIONS | `false` | Drops every collection which is being imported |
 | REPLACE_ID | `false` | Replaces `id` property with `_id` for every document during import; useful for ORMs | 
+| SET_TIMESTAMPS | `false` | Set timestamps for every document during data import |
```
[the remark this answers is not in the document]
Comment (pkosiec) [reply cli/README.md:88]: Apart from the tests, please also apply this suggestion.

If you need any help with fixing tests, let me know :-) Cheers!

[the remark this answers is not in the document]
Comment (benjaminb-) [reply cli/README.md]: Hey, this one is fixed,

Yes actually I'm struggling a little bit with the integration tests. I'm relatively new in the world of Javascript and Jest, so I'm really interested and curious to understand how you will handle that case, so yes thanks for the help!

[the remark this answers is not in the document]
Comment (pkosiec) [reply cli/README.md]: Sure, I'm on it!

**Review by pkosiec — APPROVED** [review]
LGTM
I fixed the integration tests and now everything is green.
Thank you @benjaminb- again for your contribution!

**Pull request merged**